src/main/resources/application.properties

```diff
@@ -1,3 +1,9 @@
 #TODO: Spring profile local
 spring.application.name=ask
-server.port=1337
\ No newline at end of file
+server.port=1337
+
+spring.datasource.url=jdbc:h2:mem:testdb
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=sa
+spring.datasource.password=
```

Comment:
Is this supposed to be "password"?  Try it

Reply:
No current difference when changing the password